data/hai/unfettered 0 prologue.txt

```diff
@@ -473,20 +475,20 @@ mission "Unfettered Jump Drive Trading"
 			`	<as before (run)> manage to dodge the blade but are then left defenseless when the same Unfettered closes with another one and slashes down to strike. They stop at the last possible moment, dissatisfaction clear on their face.`
 			branch fail
 				has "unfettered: flee from the blade"
-			`	"So this is how you 'fight,' Captain? You hide? You dodge? You refuse to seize our weapons, so you will not have them."`
+			`	"So this is how you 'fight'<, unfettered title>? You hide? You dodge? You refuse to seize our weapons, so you will not have them."`
 				goto fail
 			label flee
 			action
 				"unfettered: jd to sell" += 4
 				"unfettered: flee from the blade" ++
 			`	<as before (flee)> run away from the blade without any difficulty, but the Hai that threw it catches up to you in mere seconds and immobilizes you with their blade on your throat.`
-			`	"You disappoint me, Captain. In battle it is of great importance that you identify your advantages over the enemy and not show weakness by running, else they will come in for the kill. You have failed in both ways here. Never would you have been able to outrun me. All you did was show fear."`
+			`	"You disappoint me<, unfettered title>. In battle it is of great importance that you identify your advantages over the enemy and not show weakness by running, else they will come in for the kill. You have failed in both ways here. Never would you have been able to outrun me. All you did was show fear."`
 				to display
 					not "unfettered: dodged the blade"
 			label fail
 			action
 				"unfettered: blade rodeo" ++
-			`	"You really keep on disappointing me, Captain. At one time you tried to avoid a fight you could not flee from, and the other to dodge a weapon that was so readily presented to you? Maybe you truly are no warrior and let most of your subordinates do the dirty work."`
+			`	"You really keep on disappointing me<, unfettered title>. At one time you tried to avoid a fight you could not flee from, and the other to dodge a weapon that was so readily presented to you? Maybe you truly are no warrior and let most of your subordinates do the dirty work."`
```

Comment:
Either:
"At the one time you tried to avoid a fight you could not flee from, and at the other you tried"
or:
"Once you tried to avoid a fight you could not flee from, and then you tried"

Reply:
```suggestion
			`	"You really keep on disappointing me<, unfettered title>. At the one time you tried to avoid a fight you could not flee from, and at the other to dodge a weapon that was so readily presented to you? Maybe you truly are no warrior and let most of your subordinates do the dirty work."`
```
However I'm not sure but I think it's fine as is? This is sort of outside the scope of this though ig we could always do a quick fix